modules/javafx.controls/src/main/java/javafx/scene/control/Pagination.java

```diff
@@ -86,12 +87,20 @@
  * pagination.setPageFactory(new Callback&lt;Integer, Node&gt;() {
  *     &#064;Override
  *     public Node call(Integer pageIndex) {
- *         return new Label(pageIndex+1 + ". Lorem ipsum dolor sit amet,\n"
+ *         return new Label(pageIndex + 1 + ". Lorem ipsum dolor sit amet,\n"
```

Comment:
Do you think we need keep the non-lambda version of the example? It's up to you.

Reply:
Other controls have them, so I kept it.